berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -94,26 +94,31 @@ class MapMarkerDetailView: UIView {
         closeButton.widthAnchor.constraint(equalToConstant: kButtonSize).isActive = true
         closeButton.heightAnchor.constraint(equalToConstant: kButtonSize).isActive = true
         
-        verticalStack = UIStackView(axis: .vertical, distribution: .fill, spacing: kViewMargin)
-        contentView.addSubview(verticalStack)
-        verticalStack.translatesAutoresizingMaskIntoConstraints = false
-        verticalStack.topAnchor.constraint(equalTo: contentView.layoutMarginsGuide.topAnchor).isActive = true
-        verticalStack.leftAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leftAnchor).isActive = true
-        verticalStack.rightAnchor.constraint(equalTo: contentView.layoutMarginsGuide.rightAnchor).isActive = true
-        
         nameLabel = UILabel()
+        contentView.addSubview(nameLabel)
         nameLabel.numberOfLines = 0
         nameLabel.font = Font.bold(20)
         nameLabel.textColor = Color.primaryText
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
+        nameLabel.topAnchor.constraint(equalTo: contentView.layoutMarginsGuide.topAnchor, constant: -offset).isActive = true
```

Comment:
Why the `constant: -offset` instead of having it flush with the layout margins?

Reply:
Offset to match vertially align nameLabel with the "x"